concepts/basics/introduction.md

```diff
@@ -0,0 +1,111 @@
+# Introduction
+
+To set off the C++ journey we are starting with variables, function calls, and comments.
+
+## Comments
+
+Comments come in two flavors: single- and multi-line.
+Everything that comes after `//` on the same line is ignored by the compiler.
+Multi-line comments are also known as C-style comments. 
+They are surrounded by `/*` and `*/`.
+Anything that comes between these will be ignored as well.
+
+## Variables
+
+C++ is a typed language.
+All types need to be known at compile time, and so you generally need to state them explicitly.
+A variable's type cannot change.
+An integer variable with the name `years` can be declared like this:
+
+```cpp
+int years;
+```
+
+It is good practice to initialize variables upon declaration.
+C++ offers different mechanisms to do so.
+The version with the curly braces is more in line with modern C++, but the equal-sign version is also very common.
+
+```cpp
+int tomatoes{80};
+int potatoes = 40;
+```
+
+```exercism/caution
+C++ does allow using uninitialized variables.
+Until the variable is deliberately set, it is undefined and might contain anything.
+To avoid used-before-set errors and undefined behaviour it is adviseable to **always initialize**.
+```
+
+## Arithmetic Operations
+
+Arithmetic operators like `*`, `+`, or `-` can be part of an expression like `3 * 2` or `tomatoes + potatoes`. 
+
+## Updating Variables
+
+You can reassign variables, as long as they keep their type:
+
+```cpp
+tomatoes = tomatoes - 5; // tomatoes is now 75
+potatoes = (32 * 2) + 11; // potatoes is now 75 as well
+```
+
+## Functions
+
+Functions have a name, a return type and (possibly empty) parameter list.
```

Comment:
Is there a writing style in c++? If functions should be written in snake case it could be worth a mention,

Reply:
Unfortunately there is no single coding style guideline, but [quite a lot of them](https://gist.github.com/derofim/df604f2bf65a506223464e3ffd96a78a#useful-links). Personally I would opt for the [CppCoreGuidelines style](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#S-naming).  Do you thin we should include that somewhere?

Reply:
I mean it depends on, like for this example, is there an agreed-upon standard to use snake case? Or is it alright to use something like camelcase? Since in a language like Ruby, there is more or less a consensus to write methods (functions) in snake case and if it is a consensus in a language community I think it could be worth mentioning.

Reply:
> Do you thin we should include that somewhere?

You already went a bit into style in the Whitespace section. It could be expanded to explain that many coding style standards exist. I'd argue against recommending adopting any specific one though. (Or perhaps even mentioning any.)

Students who want to know about the various standards can search for them. Students who _need_ to know we cannot help: their situation dictates it for them.

I do not think there is much value in trying to adhere to any one standard on this track. I just don't see the point of doing so. Using your best judgment case by case seems good enough, less effort, and less arbitrary/quirky.

@meatball133 No, C++ is a wild west.